### ISSUE TITLE
[Feature request] - add metrics

### ISSUE THREAD

**Issue by zhernovskyi** [denysdovhan/ha-lun-misto-air#13]
Currently Lun API provides few more metrics(temp, humidity, pressure). It would be nice to have them all as well.

**Comment by denysdovhan** [comment]
Good point! This is a nice feature, and it should be relatively easy to implement.
I'll take that issue when I have spare time. If you want to speed up this feature, please, consider contributing.
